setup.py

```diff
@@ -4,6 +4,8 @@
 
 from setuptools import Extension, setup
 
+DEFLATE_REQUIRED_VERSION = ">= 1.10"
+
```

Comment:
you could add something to force using the bundled code (default: try finding and using the system lib), see how borgbackup deals with the libs in `setup*.py`.

https://github.com/borgbackup/borg/blob/1.2.0/setup.py#L49